test/integration/query-interface.test.js

```diff
@@ -454,29 +456,60 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
         },
       });
     });
+    afterEach(async function () {
+      await this.queryInterface.dropTable('hosts');
```

Comment:
Is the goal to get off the global or would you prefer to rely on it? I thought your other post suggested doing it in chunks, starting by removing `dropAllDatabase` from the global and placing it in each file. Isn't the eventual state to have each test be explicit in what it builds and tears down (like this) or do you have a different vision in mind?

Reply:
Yes you're right. I commented because it slowed the test unnecessarily but we should move towards removing `dropAllDatabase` asap so this can stay in prevision of that